portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +27,24 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private List<String> comments = new ArrayList<>();
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(comments);
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
-  }
   
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment");
-    comments.add(text);
+    long time = System.currentTimeMillis();
+
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("comment", text);
+    commentEntity.setProperty("time", time);
```

Comment:
Times are notoriously tricky to deal with in programming, and descriptive names can help avoid conversion and arithmetic errors. Two guidelines:

1. Clearly express the time's meaning/significance. In this case, we're storing the time at which a comment was created.
2. If your stored format is not a rich timestamp type, it's best to declare units.

In this case, I would suggest "createdTimeMillis".